sycl/include/CL/__spirv/spirv_types.hpp

```diff
@@ -126,7 +126,9 @@ struct ConstantPipeStorage {
 };
 
 // Arbitrary precision integer type
+namespace _spv {
```

Comment:
I think the right name for the namespace is `__spv`. I also agree with Ronan that the issue this patch fixes is not specific to triSYCL project, so it should be fixed in upstream.
```suggestion
namespace __spv {
```